server/manager/src/main/java/org/apache/accumulo/manager/ManagerClientServiceHandler.java

```diff
@@ -397,21 +399,25 @@ private void alterNamespaceProperty(TCredentials c, String namespace, String pro
 
     try {
       if (value == null) {
-        NamespacePropUtil.removeNamespaceProperty(manager.getContext(), namespaceId, property);
+        NamespacePropUtil.factory().removeProperties(manager.getContext(), namespaceId,
+            List.of(property));
       } else {
-        NamespacePropUtil.setNamespaceProperty(manager.getContext(), namespaceId, property, value);
+        NamespacePropUtil.factory().setProperties(manager.getContext(), namespaceId,
+            Map.of(property, value));
+      }
+    } catch (PropStoreException ex) {
+      if (ex.getCause() instanceof KeeperException.NoNodeException) {
```

Comment:
Would you expect the ZK exception to always be one level deep as the cause?  If not may need to loop/recurse to search for a ZK cause.

Reply:
I think they should always be one level deep for this case. If it wasn't, we might want to handle it completely differently.

Reply:
The PropStoreException was removed and IllegalStateException is used instead